tetris_3.py

```diff
@@ -27,8 +27,7 @@
 ]
```

Comment:
by the way, I think I made a mistake here with the definition of the problem. it's not necessary to have nested arrays for the definition of the pieces, it should be just:
```
PIECES = [
  ['HHHH'],
  ['ZZ',
   ' ZZ'],
  [' SS',
   'SS'],
...
]
```
please feel free to change it if it does make sense. my apologies